be/src/vec/columns/column_array.cpp

```diff
@@ -235,7 +235,13 @@ void ColumnArray::insert_from(const IColumn& src_, size_t n) {
     size_t size = src.size_at(n);
     size_t offset = src.offset_at(n);
 
-    get_data().insert_range_from(src.get_data(), offset, size);
+    // Note: here we should process the case of 'Nullable(Array(NotNullable(nest)))'
+    if (get_data().is_nullable() && !src.get_data().is_nullable()) {
+        reinterpret_cast<ColumnNullable*>(&get_data())
+                ->insert_range_from_not_nullable(src.get_data(), offset, size);
+    } else {
```

Comment:
add DCHEC for `!get_data().is_nullable() && src.get_data().is_nullable()` ?

Reply:
if add DCHEC to replace, the insert_from() will just support the only one case. It is unreasonable.

Reply:
done